ballerina/tests/company_client.bal

```diff
@@ -70,12 +70,12 @@ client class CompanyClient {
         }
     }
 
-    remote function update(record {} 'object, map<anydata> filter) returns Error? {
-        _ = check self.persistClient.runUpdateQuery('object, filter);
+    remote function update(Company 'object) returns Error? {
+        _ = check self.persistClient.runUpdateQuery('object);
     }
 
-    remote function delete(map<anydata> filter) returns Error? {
-        _ = check self.persistClient.runDeleteQuery(filter);
+    remote function delete(Company 'object) returns Error? {
```

Comment:
Remove filter param and change type of 1st param to entityName from update() method